Server/Ebenezer/Define.h

```
@@ -117,7 +117,6 @@ enum e_Exec
 	EXEC_GIVE_PPCARD_ITEM					= 0x0D,
 	EXEC_SAVE_EVENT							= 0x0E,
 	EXEC_PROMOTE_USER						= 0x0F,
-	EXEC_GIVE_PROMOTION_QUEST				= 0x10,
```

The define should remain here.  It just shouldn't eat the exception handling in the switch cases.